jena-integration-tests/src/test/java/org/apache/jena/http/AuthBearerTestLib.java

```diff
@@ -81,7 +80,7 @@ public static String generateTestToken(String user) {
     private static String enc64(String x) {
         byte[] bytes = x.getBytes(StandardCharsets.UTF_8);
         // URL encoding, no padding, no chunking line breaks.
-        String s = org.apache.commons.codec.binary.Base64.encodeBase64URLSafeString(bytes);
+        String s = Base64.getUrlEncoder().encodeToString(bytes);
```

Comment:
OAuth at least is not `base64url`, but is `base64`.
This bug is concealed by using the general decoder. If changing the decoder to only `base64`, this test needs to be fixed.

```suggestion
        String s = Base64.getEncoder().encodeToString(bytes);
```